studybuddies/src/app/student/class-list-student/class-list-student.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-class-list-student',
+  templateUrl: './class-list-student.component.html',
+  styleUrls: ['./class-list-student.component.css']
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7155399)
```suggestion
  styleUrls: ["./class-list-student.component.css"]
```